bindata/assets/alerts/cpu-utilization-sno.yaml

```diff
@@ -0,0 +1,82 @@
+kind: PrometheusRule
+metadata:
+  name: cpu-utilization
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+    - name: control-plane-cpu-utilization
+      rules:
+        - alert: HighOverallControlPlaneCPU
+          annotations:
+            summary: >-
+              CPU utilization across plane pods is high; a control plane pods outage may cause a cascading failure;
+              check control plane logs for possible issues and/or add more CPU and modify alert threshold.
+            runbook_url: PASS
+            description: >-
+              By default, this alert is triggered if all control plane pods are using more that 3 CPUs(3000 millicores).
+              Usually, when they start to utilize more CPU, this means that there are might be issues at them.
+              It's recommended to check pods and their logs to verify that there are not issues.
+              If all is ok, then increase the CPU on the node and increase alert threshold.
+          expr: |
+            sum(
+              rate(
+                container_cpu_system_seconds_total{namespace=~"openshift-etcd|openshift-kube-apiserver|openshift-kube-controller-manager|openshift-kube-scheduler|openshift-apiserver|openshift-controller-manager|openshift-authentication",image!=""
+                }[5m]
+              )
+            )
+            > 3
+          for: 10m
+          labels:
+            namespace: openshift-kube-apiserver
+            severity: warning
+        - alert: ExtremelyHighIndividualControlPlaneCPU
+          annotations:
+            summary: >-
+              CPU utilization on a control plane pods is very high, more CPU pressure is likely to cause a failover; increase available CPU.
+            runbook_url: PASS
+            description: >-
+              Extreme CPU pressure can cause slow serialization and poor performance from the kube-apiserver and etcd.
+              When this happens, there is a risk of clients seeing non-responsive API requests which are issued again
+              causing even more CPU pressure.
+              It can also cause failing liveness probes due to slow etcd responsiveness on the backend.
+              If one kube-apiserver fails under this condition, chances are you will experience a cascade as the remaining
+              kube-apiservers are also under-provisioned.
+              If all is ok, then increase the CPU on the node and increase alert threshold.
+          expr: |
+            sum(
+              rate(
+                container_cpu_system_seconds_total{namespace=~"openshift-etcd|openshift-kube-apiserver|openshift-kube-controller-manager|openshift-kube-scheduler|openshift-apiserver|openshift-controller-manager|openshift-authentication",image!=""
+                }[5m]
+              )
+            )
+            > 3.4
+          for: 5m
+          labels:
+            namespace: openshift-kube-apiserver
+            severity: alert
+        - alert: ExtremelyHighIndividualControlPlaneCPU
```

Comment:
rather than having a new alert name, I'd keep the same name and leverage different severities

Reply:
In case of SNO clusters, you probably want only 2 `ExtremelyHighIndividualControlPlaneCPU` alerting rules (one warning and one critical). The `HighOverallControlPlaneCPU` alerting rule which aggregates across all control-plane nodes  isn't relevant since 1 cluster = 1 node.